WordPress/Classes/Utility/TaggedManagedObjectID.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+
+/// `TaggedManagedObjectID` is a `NSManagedObjectID` wrapper that also contains the model type of the `NSManagedObjectID`.
```

Comment:
```suggestion
/// `TaggedManagedObjectID` is an `NSManagedObjectID` wrapper that also contains the model type of the `NSManagedObjectID`.
```

Since the "N" in "NSManaged" is pronounced with its name/sound, "en", it should be preceded by the article "an".

Here's a [Quora source](https://qr.ae/pyzs7B) and [ChatGPT's take](https://chat.openai.com/share/e2844bcd-990e-4c65-b855-a8f8aa724477). Neither is really authoritative, but I think it's enough backing for the grammar rule.